### ISSUE TITLE
Very interesting work!

### ISSUE THREAD

**Issue by nixtoshi** [lchen001/LLMDrift#4]
What do you think might be the **cause** of LLM drift?

I had a suspicion that this was the case, because GPT4 seemed to be learning in real time (or something similar), or OpenAI stealth updating their model called "GPT4", waiting for a certain large number of new parameters before calling it "GPT5". But these are just speculations, I don't know the cause.

If it's one of these 2 though, I think the new feedback that GPT4 is getting might be more inaccurate that accurate, and there might also be malicious feedback being incorporated into the model.

I think your paper also shows that "freezing" LLMs and keeping their **different versions** will have more utility in certain applications where replicability and stability is needed at the expense of making it acquire new knowledge.

**Comment by YashDagade** [comment]
I agree. Freezing model will be an important step so will creating a comprehensive evaluation task.  I also think that updates rolled out should be named versions. For these models to mimic reproducibility, we first need to ensure that for every minor change in the model, they name the release.
